packages/ui-components/src/components/SpinningLoader.tsx

```diff
@@ -0,0 +1,20 @@
+/*
+ * Tupaia
+ *  Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
```

Comment:
I think that this is a different component to the other loading spinners in ui-components but there are a few now and it would be easy for duplicates to creep in so would you mind putting them all in a folder.
- FullPageLoader
- LoadingContainer
- SpinningLoader